lib/PuppeteerSharp/Page.cs

```diff
@@ -967,6 +967,11 @@ public async Task ScreenshotAsync(string file, ScreenshotOptions options)
             if (!options.Type.HasValue)
             {
                 options.Type = ScreenshotOptions.GetScreenshotTypeFromFile(file);
+
+                if (options.Type == ScreenshotType.Jpeg && !options.Quality.HasValue)
+                {
+                    options.Quality = 90;
```

Comment:
@kblok I am ok with that.  I picked 90 because it's the default on Paint.net.  And on Photoshop the default is 10 out of 12.  So seemed like a consensus.

Reply:
Good explanation. Let’s leave it 🙂